app-misc/jigdo/jigdo-0.7.3.recipe

```diff
@@ -0,0 +1,73 @@
+SUMMARY="Ease the distribution of files"
+DESCRIPTION="Jigsaw Download, or short jigdo, is a tool designed to ease the distribution \
```

Comment:
Oops, the 80 chars limit :)
Here it is, rewrapped, for your convenience.
```
DESCRIPTION="Jigsaw Download, or short jigdo, is a tool designed to ease the \
distribution of very large files over the internet, for example CD or DVD \
images. Its aim is to make downloading the images as easy for users as a click \
on a direct download link in a browser, while avoiding all the problems that \
server administrators have with hosting such large files."
```

BTW, the `SUMMARY` is OK as is, after all, although I would prefer the variant I suggested.